### ISSUE TITLE
colaboração?

### ISSUE THREAD

**Issue by tofanelli** [pmanizio/linksuteis#11]
Opa... tranquilo?

Eu vi que eu tenho uns links legais que você não tem aqui, aceita que te mande os links para poder aumentar a lista ainda mais?

valeu =)

**Comment by pamelama** [comment]
Claro, fique a vontade para contribuir. Desculpe pela demora na resposta, em breve voltarei a me dedicar ao projeto :)
